public/app/features/logs/utils.ts

```diff
@@ -63,6 +64,37 @@ export function getLogLevel(line: string): LogLevel {
       }
     }
   }
+  // z/OS syslog messages often use a single-letter severity at the end of the message
+  // e.g. "... MSGID I" or "... (I)" where I=informational, W=warning, E=error, S=severe
+  // Try to detect a trailing severity letter if no other token matched earlier in the line.
+  if (level === LogLevel.unknown) {
+    if (config.featureToggles['zosSyslogLevelMapping']) {
+      const zosRegexp = /(?:\b|\()([IWEFCS])(?:\b|\))/i; // include common letters used in syslog variants
```

Comment:
This needs to be reverted because the chances for false positives is too high. The mere presence of a character in a log line is not a guarantee of the inferred level.